### PR TITLE
Implemented: code to update runTime and make the runTime button clickable(#24b0y61)

### PR DESCRIPTION
**Pull request by ymaheshwari1**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

File: src/views/InitialLoad.vue [hunk 1]
```diff
@@ -51,13 +52,14 @@
                 <ion-label slot="end">{{ job?.lastUpdatedStamp ? getTime(job.lastUpdatedStamp) : $t('No previous occurrence') }}</ion-label>
               </ion-item>
 
-              <ion-item>
+              <ion-item id="open-product-run-time-modal" button>
```
Comment (adityasharma7) [inline src/views/InitialLoad.vue:55]:
```suggestion
              <ion-item id="product-run-time-modal" button>
```

File: src/views/InitialLoad.vue [hunk 2]
```diff
@@ -82,13 +85,14 @@
                 <ion-label slot="end">{{ job?.lastUpdatedStamp ? getTime(job.lastUpdatedStamp) : $t('No previous occurrence') }}</ion-label>
               </ion-item>
 
-              <ion-item>
+              <ion-item id="open-order-run-time-modal" button>
```
Comment (adityasharma7) [inline src/views/InitialLoad.vue:88]:
```suggestion
              <ion-item id="order-run-time-modal" button>
```

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/InitialLoad.vue (hunk 1 above)
Comment (ymaheshwari1) [reply src/views/InitialLoad.vue]: Improved the id for product card.

File: src/views/InitialLoad.vue (hunk 2 above)
Comment (ymaheshwari1) [reply src/views/InitialLoad.vue]: Improved the id for the order card.

**Pull request merged**